src/components/Button/Button.tsx

```diff
@@ -178,6 +183,10 @@ export const ButtonPurple = styled(Base)`
   &:active {
     background-color: ${({ theme }) => darken(0.1, theme.dark2)};
   }
+  &:disabled {
```

Comment:
Maybe we can add &:disabled styles to `Base`

```
 &:disabled {
    opacity: 50%;
    cursor: not-allowed;
  }
```

Reply:
The things is, there are so many different behaviours to disabled Buttons, I don't know if I can break something if I move it to `Base`. Maybe we could have an issue to clean this Button file and have better components. Will see what I can do.